app/controllers/items/detail.js

```diff
@@ -411,6 +413,37 @@ export default GoodcityController.extend(
         );
       },
 
+      setPkgDescriptionLang(langauge) {
+        this.set("selectedDescriptionLanguage", langauge);
+      },
+
+      async updateComment(name, val) {
+        const item = this.get("item");
+        if (item.changedAttributes()[name]) {
+          const params = { package: { [name]: val } };
+          await this.runTask(async () => {
+            await this.get("packageService").updatePackage(item, params);
+          }, ERROR_STRATEGIES.MODAL);
+        }
+      },
+
+      async updateDescription(name, val) {
+        const item = this.get("item");
+        if (item.changedAttributes()[name]) {
+          const params = { package: { [name]: val } };
+          await this.runTask(async () => {
+            await this.get("packageService").updatePackage(item, params);
+          }, ERROR_STRATEGIES.MODAL);
+        }
+        this.send("setShowDescSuggestion", false);
+      },
```

Comment:
Those 2 methods seem to be doing pretty much the same thing

```suggestion
      async updateAttribute(prop, val, cb = _.noop) {
        const item = this.get("item");
        if (item.changedAttributes()[prop]) {
          const params = { package: { [prop]: val } };
          await this.runTask(async () => {
            await this.get("packageService").updatePackage(item, params);
            cb();
          }, ERROR_STRATEGIES.MODAL);
        }
      },
      
      updateComment(val) {
         return this.updateAttribute('comment', val);
      },
      
      updateDescription(val) {
         return this.updateAttribute('description', val, () => {
             this.send("setShowDescSuggestion", false);
         });
      },
```